huxley/logging/LoggingEmailBackend.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
+# Use of this source code is governed by a BSD License (see LICENSE).
+
+import json, logging
+
+from django.core.mail.backends import dummy, smtp
+from django.db import transaction
+
+from smtplib import SMTPException
+
+
+class DevLoggingEmailBackend(dummy.EmailBackend):
+
+    def send_messages(self, email_messages):
+        logger = logging.getLogger('huxley.api')
+        for email in email_messages:
+            recipients = email.to[0] if len(email.to) == 1 else ', '.join(email.to)
```

Comment:
Just the join will work, it won't add the delimiter if the length of the list is just one:
```python
recipients = ', '.join(email.to)
```